python/dgl/graphbolt/minibatch.py

```diff
@@ -474,6 +474,35 @@ def apply_to(x, device):
         return self
 
 
+    def to_pyg_adapter(self, device):
+        from torch_geometric.data import Data
+
+        def construct_edge_index(subgraph, device):
+            csc_matrix = subgraph.sampled_csc
+            indices = csc_matrix.indices.to(device)
+            indptr = csc_matrix.indptr.to(device)
+            row_indices = torch.arange(
+                len(indptr) - 1, device=device
+            ).repeat_interleave(indptr.diff())
```

Comment:
```suggestion
            row_indices = gb.expand_indptr(indptr, dtype=indices.dtype, output_size=len(indices))
```
After updating with the master, replace as above. PR #6871 was just merged into master.

Reply:
i just updated with the master branch. It seems still error

Traceback (most recent call last):
  File "/home/ubuntu/dgl/tests/python/pytorch/graphbolt/impl/test_minibatch.py", line 904, in <module>
    test_to_pyg_adapter()
  File "/home/ubuntu/dgl/tests/python/pytorch/graphbolt/impl/test_minibatch.py", line 872, in test_to_pyg_adapter
    pyg_data = test_minibatch.to_pyg_adapter(device=torch.device("cpu"))
  File "/home/ubuntu/dgl/python/dgl/graphbolt/minibatch.py", line 532, in to_pyg_adapter
    edge_index = construct_edge_index(subgraph, device)
  File "/home/ubuntu/dgl/python/dgl/graphbolt/minibatch.py", line 525, in construct_edge_index
    row_indices = expand_indptr(indptr, dtype=indices.dtype, output_size=len(indices))
  File "/home/ubuntu/dgl/python/dgl/graphbolt/base.py", line 100, in expand_indptr
    return torch.ops.graphbolt.expand_indptr(
  File "/opt/conda/envs/dgl-dev-gpu-121/lib/python3.8/site-packages/torch/_ops.py", line 761, in __getattr__
    raise AttributeError(
AttributeError: '_OpNamespace' 'graphbolt' object has no attribute 'expand_indptr'

Reply:
thank you @yxy235  just solved it